src/backend/tests/seed.ts

```diff
@@ -25,8 +25,21 @@ export const seed = async (db: KyselyDatabaseInstance) => {
     .returningAll()
     .executeTakeFirstOrThrow();
 
+  //! For tests, is it better to insert into database this para or have a seed of this para already in the db?
```

Comment:
why can't we use the existing para above?

Reply:
In the test, we're using the authentication as `para`. In order to use the existing para above, we have to authenticate as `admin` right?

Reply:
sorry, not understanding

passing `para` to the test fixture authenticates as the above para:

![Screenshot 2023-06-24 at 9 02 02 PM](https://github.com/sfbrigade/compass/assets/7410405/02edd2ff-c4f3-4d03-8a83-77a487bd1bd3)

(we should probably change this to `staff` to match the role but that can be a separate PR)